x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/SemanticSearchAction.java

```diff
@@ -43,17 +46,15 @@
 
 public class SemanticSearchAction extends ActionType<SemanticSearchAction.Response> {
 
-    // TODO what should this be called? If this becomes an indices action
-    // change the transport code to run _infer as ML_ORIGIN
-    public static final String NAME = "cluster:monitor/xpack/ml/semantic_search";
+    public static final String NAME = "indices:data/read/semantic_search";
```

Comment:
Since the original change has not been released, changing it again has no BWC concern. I think this is the case. But  would like to double confirm.

Reply:
Correct this action is unreleased we are targeting the 8.6 release